### PR TITLE
Proposal to show tx on success plus --json option

### LINKED ISSUE #51: [FEATURE] Return send ZCN txn hash on success
In 0wallet the user does not get a transaction hash back when they send ZCN - only "success" this makes it difficult for the user to verify the transaction later, or keep the id for their records.

image

Acceptance criteria:
Return "Send ZCN success with txn ID: [{{id}}]"

### PR DESCRIPTION
**Pull request by sculptex**
Combined request to solve https://github.com/0chain/zwalletcli/issues/51 and add --json option @stewartie4 

Invalid json response will indicate error unless we format all responses as json when requested

A brief description of the changes in this PR:

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zwalletcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- system_test:  https://github.com/0chain/system_test/pull/271

### CODE REVIEW

File: cmd/send.go [hunk 1]
```diff
@@ -56,7 +59,15 @@ var sendcmd = &cobra.Command{
...
-				fmt.Println("Send tokens success")
+				if doJSON {
+					j := map[string]string {
+		  			"status": "success",
+		  			"tx": txn.Hash(),
+		  			"nonce": strconv.FormatInt(txn.GetTransactionNonce(),10) }
+					util.PrintJSON(j)
+					return
+				}
+				fmt.Println("Send tokens success:", txn.Hash())
```
Comment (cnlangzi) [inline cmd/send.go]: revert it to fix system_test.
```suggestion
fmt.Println("Send tokens success:")
```

**Review by cnlangzi — CHANGES_REQUESTED** [review]

File: cmd/send.go [hunk 2]
```diff
@@ -56,7 +59,15 @@ var sendcmd = &cobra.Command{
...
-				fmt.Println("Send tokens success")
+				if doJSON {
+					j := map[string]string {
+		  			"status": "success",
+		  			"tx": txn.Hash(),
+		  			"nonce": strconv.FormatInt(txn.GetTransactionNonce(),10) }
+					util.PrintJSON(j)
+					return
+				}
+fmt.Println("Send tokens success:")
```
Comment (cnlangzi) [inline cmd/send.go]:
```suggestion
				fmt.Println("Send tokens success")
```

**Review by cnlangzi — CHANGES_REQUESTED** [review]

**Review by cnlangzi — APPROVED** [review]
LGTM

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/print_hash_on_send  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | sculptex-sendtxjson  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/print_hash_on_send  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | sculptex-sendtxjson  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM

**Pull request merged**